core/io/config_file.cpp

```diff
@@ -111,7 +111,7 @@ void ConfigFile::get_sections(List<String> *r_sections) const {
 }
 void ConfigFile::get_section_keys(const String &p_section, List<String> *r_keys) const {
 
-	ERR_FAIL_COND(!values.has(p_section));
+	ERR_FAIL_COND_MSG(!values.has(p_section), "Cannot get from config file section '" + p_section + "'.");
```

Comment:
I'd be more specific, `"Cannont get keys from nonexistent section '" + p_section "'."`.